.github/workflows/semantic-release.yml

```
@@ -27,12 +29,30 @@ jobs:
       pull-requests: write
+    
+    # Add a step to check for cache skip flag in commit message
+    env:
+      SKIP_CACHE_RESTORE: false
     steps:
       - name: Checkout
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
           persist-credentials: false
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)